deployment/build.windows.amd64

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+#= Windows 7+ amd64 (x64) .zip
+
+set -o errexit
+set -o xtrace
+
+# Version variables
+NSSM_VERSION="nssm-2.24-101-g897c7ad"
+NSSM_URL="http://files.evilt.win/nssm/${NSSM_VERSION}.zip"
```

Comment:
I think we now store it somewhere at our own repo.jellyfin.org, right @dkanada ?

Reply:
ibid.